refactor(vitest): rename installQuasar in installQuasarPlugin

ilmalte opened the pull request:
Renamed installQuasarPlugin in installQuasar in accordance with the interface exported by the package @quasar/quasar-app-extension-testing-unit-vitest

**What kind of change does this PR introduce?**

- [x] Documentation

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No


**Other information:**
Seems like vitest export is different from Jest and Cypress. If there is no technical reason behind, this could be misleading 😄

IlCallo commented:
Seems like the problem here is the other way around: the helper function should be named `installQuasarPlugin` to be coherent with Cypress and Jest implementation of the same helper, since we'll need to abstract it into a package shared by all testing harnesses in the future

Could you perform the change? I'll then bump deps and release a new version under the beta tag

ilmalte commented:
What kind of change does this PR introduce? (check at least one)

- [ ] Bugfix
- [ ]  New test runner
- [ ]  Documentation
- [ ]  Feature
- [ ]  Code style update
- [x]  Refactor
- [ ]  Build-related changes
- [ ]  Other, please describe:

Does this PR introduce a breaking change? (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and migration path for existing applications:

The vitest API after this PR will change. The export function will no more be _installQuasar_ but _installQuasarPlugin_ in order to be coherent with Cypress and Jest. I renamed it and introduced this breaking change, but we could still consider using a JSDoc comment to mark the old, deprecated code.

IlCallo commented:
Being in alpha phase, I feel it's ok to add a breaking change at this stage without deprecation
Would be worse if we did it in beta/stable phase

Thanks for contributing!
